packages/contracts/hardhat.config.ts

```diff
@@ -75,6 +75,13 @@ const config: HardhatUserConfig = {
       saveDeployments: true,
       deploy: ['./deploy/L1'],
     },
+    e2eL1: {
+      live: false,
+      url: 'http://localhost:8545',
+      accounts: [PRIVATE_KEY_DEPLOYER_DEVNET],
+      saveDeployments: true,
+      deploy: ['./deploy/L1'],
+    },
```

Comment:
Here also how about just using `devnetL1` network since it's all same?

Reply:
I think it's not possible since hardhat saves deployments at `packages/contracts/deployments` folder with the network name, as well as reading the deploy config with the network name.